crates/bevy_render/src/camera/bundle.rs

```diff
@@ -75,6 +75,22 @@ pub struct OrthographicCameraBundle {
 }
 
 impl OrthographicCameraBundle {
+    /// Create an orthographic projection camera to render 2D content.
+    ///
+    /// The projection creates a camera space where X points to the right of the screen,
+    /// Y points to the top of the screen, and Z points out of the screen (backward),
+    /// forming a right-handed coordinate system. The center of the screen is at X=0 and
+    /// Y=0.
+    ///
+    /// The scaling mode is the default for [`OrthographicProjection`], resulting in
```

Comment:
> The scaling mode is the default for

As a reader, I'm not immediately sure what this refers to. Can this be configured somewhere?

Reply:
<img width="944" alt="ortho" src="https://user-images.githubusercontent.com/6532395/147504270-60f23837-e1ae-49b9-8572-57ea3a8e3dac.png">